collector/settings.mdx

```diff
@@ -537,6 +547,15 @@ talking to pganalyze support.
       <td>Experimental: name of docker container to collect logs from using <code>docker logs -t</code>.
       This requires that the collector runs on the Docker host.</td>
     </tr>
+    <tr>
+      <td>disable_citus_schema_stats (<code>DISABLE_CITUS_SCHEMA_STATS</code>)</td>
+      <td>[none]</td>
+      <td>
+        If using the Citus extension in your database, turn off the collection of statistics for distributed indexes
+        and tables. For very large schemas, this collection can be a performance problem. When using this option it's
```

Comment:
Maybe "can be a performance problem" => "can error out due to timeouts or locks"

I think our statement timeout makes sure this isn't an actual performance problem, but if you have many tables its likely to either run into timeout issues (i.e. it takes too long), or locking problems because of how this is implemented in Citus.